silverback/application.py

```diff
@@ -88,6 +107,92 @@ def __init__(self, settings: Settings | None = None):
             f"{signer_str}{start_block_str}{new_block_timeout_str}"
         )
 
+        # NOTE: The runner needs to know the set of things that the app is tracking as a parameter
+        # NOTE: We also need to know the defaults in case the parameters are not in the backup
+        self.parameter_defaults: dict[str, ScalarType | None] = dict()
+
+        self._create_system_startup_task()
+        self._create_system_backup_task()
```

Comment:
thought: Wondering if registering tasks on the runner is going to cause some issues.  Everything else requires the user app to call as decorators.

Reply:
In both cases, the app is doing the primary task registration. The runner will call a subset of those tasks using the same app (containing the same task definitions as loading the app's module will perform the same registration steps)

Ideally we could actually make this more stateless by having a defined system task that returns the worker's task configuration that the runner can just read. All the tasks we allow a runner to run currently have a defined structure, so we know how to send them args and handle their results (We do not support arbitrary task defs, although of course a task in TaskIQ can kick off executing another task as a consequence) The only potential problem there is that an app cannot change it's runtime configuration of tasks after it starts (which may be desirable, but can probably be acheived in other ways)